common/src/com/intellij/plugins/haxe/config/sdk/impl/HaxeSdkAdditionalDataBaseImpl.java

```diff
@@ -34,8 +34,8 @@ public HaxeSdkAdditionalDataBaseImpl() {
   }
 
   public HaxeSdkAdditionalDataBaseImpl(String homePath, String version) {
-    this.homePath = homePath;
-    this.version = version;
+    this.homePath = null == homePath ? "" : homePath;
+    this.version  = null == version  ? "" : version;
```

Comment:
For this file, we were getting NPEs on access after the values were set to null.  Rather than change the calling code to deal with NPEs, I just mapped it to an empty string -- calling code was already testing for that.